src/main/java/study/step3/ResultView.java

```diff
@@ -0,0 +1,28 @@
+package study.step3;
+
+import study.step3.domain.Car;
+
+import java.util.List;
+
+public class ResultView {
+    private static final char POSITION_MARK = '-';
+    public void printCars(List<Car> cars) {
+        for (Car car : cars) {
+            System.out.println(getPositionStr(car));
+        }
+        System.out.println();
+    }
+
+    public void printInit() {
+        System.out.println("\n실행 결과");
+    }
+    private String getPositionStr(Car car) {
+        StringBuilder positionStr = new StringBuilder();
+        for (int i = 0; i < car.getPosition(); i++) {
+            positionStr.append(POSITION_MARK);
+        }
+        return positionStr.toString();
+
+    }
```

Comment:
```suggestion
    private static final String POSITION_MARK = "-";
    public void printCars(List<Car> cars) {
        for (Car car : cars) {
            System.out.println(getPositionStr(car));
        }
        System.out.println();
    }

    public void printInit() {
        System.out.println("\n실행 결과");
    }
    private String getPositionStr(Car car) {
        return POSITION_MARK.repeat(car.getPosition());
    }
```
String을 사용하면 더욱더 간단하게 표현할 수 있을 것 같아요 😄